tool/stats_lib.dart

```diff
@@ -74,10 +74,10 @@ class CommonMarkTestCase {
   factory CommonMarkTestCase.fromJson(Map<String, dynamic> json) {
     return CommonMarkTestCase(
         json['example'] as int,
-        json['section'] as String,
+        json['section'] as String /*!*/,
```

Comment:
This (and below) seem really weird. `as String` should completely imply "not null". `!` shouldn't me needed!

Reply:
I guess when migration is done these will be eliminated. This is just to hint the tool?

Reply:
Yeah, the migration tool sometimes decides that that should be a nullable string.